nimbus/examples/experiment.rs

```diff
@@ -325,6 +328,7 @@ fn main() -> Result<()> {
                     .get_all_experiments()
                     .expect("can't fetch experiments!?")
                 {
+                    println!("{}", exp.slug.clone());
```

Comment:
Is this leftover by accident? If not, then adding a bit more context to what's printed would be good (and I'm surprised the clone is necessary?)

Reply:
Just some debug garbage, nothing to see here...   Just forgot to remove it.